### ISSUE TITLE
关于在culane数据集上训练参数设置问题

### ISSUE THREAD

**Issue by safehumeng** [cfzd/Ultra-Fast-Lane-Detection#147]
麻烦问一下，在默认设置里面我发现参数设定上基本和论文中提到的不一样，请问这样设定是不是效果更好呢。
以下是几个不一样的地方：
1. 论文中提到的culane数据集的高度是540，是在260到530中步长为10取row_anchor，项目中是在249到589中步长为20取row_anchor（最后折合到288上了）。griding_num设定的是200
2. sim_loss_w和shp_loss_w都设置为0了
3. optimizer、learning_rate、scheduler

**Comment by cfzd** [comment]
@safehumeng 
因为这个代码是重新实现的一个版本，所以我们调整了部分的setting。我还是推荐你使用现在的setting，因为我在这个重新实现的版本上进行过一些测试得到的setting。当然你可以使用别的setting，或许也可以取得更好的效果，但这个我就不能保证了。

**Comment by zsh4614** [comment]
culane中sim_loss_w和shp_loss_w都设置为0的依据是什么?我设置为其他好像不行......

**Comment by zsh4614** [comment]
是不是因为相邻两个row_anchor采样间距太大?

**Comment by safehumeng** [comment]
> culane中sim_loss_w和shp_loss_w都设置为0的依据是什么?我设置为其他好像不行......

您训练效果怎样呢，分享一下呢
